### PR TITLE
Added new dsl-detects

### PR DESCRIPTION
**Pull request by D1nko**
Added new DSL-detects which include:
Tango REST Misconfiguration - it's a misconfiguration in tango REST server, which includes public access to critical administration functions with hard-coded credentials.
Apache Tomcat Misconfiguration - it's a misconfiguration in Apache Tomcat server, which includes public access to critical administrative functions.
Insecure cross-domain policy - misconfiguration in crossdomain.xml/clientaccesspolicy.xml files.
CRLF - is's a CRLF header injection.
Header origin/referer reflect - it detects misconfiguration in Access-Control-Allow-Origin.

### CODE REVIEW

File: crlf.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - "%0d%0ax-crlf-header: STR_MARKER"
+    - "%0ax-crlf-header: STR_MARKER"
+  - method:
+    - postfix
+detect:
+  - response:
+    - headers:
+      - 'x-crlf-header': 'STR_MARKER'
+meta-info:
+  - type: info
```
Comment (promix17) [inline crlf.yaml]: crlf

File: apache-tomcat-misconfiguration.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - into: URI
+  - payload:
+    - 'manager/html/'
+    - 'examples/'
+    - 'examples/jsp/'
+  - method:
+    - replace
+detect:
+  - response:
+    - body: '<title>401 Unauthorized<\/title>'
```
Comment (promix17) [inline apache-tomcat-misconfiguration.yaml]: Эта строчка не относится к Tomcat

File: Insecure-cross-domain-policy.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+generate:
+  - into: URI
+  - payload:
+    - 'crossdomain.xml'
+    - 'clientaccesspolicy.xml'
+  - method:
+    - replace
+detect:
+  - response:
+    - body: '<allow-access-from domain="\*"'
```
Comment (promix17) [inline Insecure-cross-domain-policy.yaml]: Добавить произвольные сивмолы в регулярку до '\*', чтобы ловить случаи типа:
"vk.\*"
например <allow-access-from domain=".\*\\\*"

File: crlf.yaml (hunk 1 above)
Comment (D1nko) [reply crlf.yaml]: done

File: apache-tomcat-misconfiguration.yaml (hunk 2 above)
Comment (D1nko) [reply apache-tomcat-misconfiguration.yaml]: done

File: Insecure-cross-domain-policy.yaml (hunk 3 above)
Comment (D1nko) [reply Insecure-cross-domain-policy.yaml]: done

**Pull request merged**